client_plugin/Makefile

```diff
@@ -319,6 +322,8 @@ deploy-vm-test:
 	$(DEPLOY_VM_TEST_SH) "$(VMS)" $(BIN) $(SCRIPTS)
 deploy-vm-plugin:
 	$(DEPLOY_VM_PLUGIN_SH) "$(VM2)" $(PLUGIN_BIN) $(MANAGED_PLUGIN_LOC) $(SCRIPTS) $(DOCKER_HUB_REPO) $(VERSION_TAG) $(EXTRA_TAG)
+deploy-plugin-windows-vm:
```

Comment:
> https://github.com/vmware/docker-volume-vsphere/pull/1667#discussion_r130504205

My suggestion is to use `deploy-plugin-windows-vm` and rename `deploy-vm-plugin` => `deploy-plugin` ... reason behind this to increase readability. Moreover `deply-vm-plugin` doesn't sound intuitive ... 

@shaominchen or @msterin: If you agree with the above reasoning then we can stay with the name what @venilnoronha has proposed otherwise renaming as per the comment.

Looking forward to your thoughts.

Thanks!